config/routes.rb

```diff
@@ -0,0 +1,22 @@
+Rails.application.routes.draw do
+
+  root to: "homepages#index"
+
+  resources :votes
```

Comment:
The only route that is actually used in VotesController is create so all the other routes shouldn't be included here.